src/components/common/Actions.vue

```diff
@@ -122,7 +122,7 @@ export default class Actions extends Mixins(FilingTemplateMixin, LegalApiMixin)
   private async onClickSaveResume (): Promise<void> {
     this.setIsSavingResuming(true)
     const filing = await this.buildFiling()
-    await this.saveFiling(filing, true)
+    await this.saveFiling(filing)
     this.setIsSavingResuming(false)
     window.location.assign(this.authUrl)
```

Comment:
Don't redirect if the save failed :)

Reply:
kk, will do.

Reply:
Assumption is that we'd `return` inside the catch block.